google-cloud-clients/google-cloud-core/src/test/java/com/google/cloud/DateTest.java

```diff
@@ -71,6 +72,27 @@ public void serialization() {
     reserializeAndAssert(Date.fromYearMonthDay(2017, 4, 20));
   }
 
+  @Test
+  public void testToDate() {
+    Date date = Date.parseDate("2016-09-18");
+    java.util.Date date1 = Date.toDate(date);
+    Calendar cal = Calendar.getInstance();
+    cal.setTime(date1);
+    assertThat(cal.get(Calendar.YEAR)).isEqualTo(2016);
```

Comment:
why cant you assert date with date1 directly?

Reply:
Java Utils Date has methods getYear, getMonth and getDayofMonth but its deprecated

Reply:
but you can still assert whole object directly know?